02_ShowUser.php

```diff
@@ -12,97 +12,103 @@
+	  
+		 <!-- Float links to the right -->
+					<div class="w3-right" >
+					  <a href="indexV1.php" class="w3-bar-item w3-button w3-xxlarge fa fa-home"></a>
+					  <a href="playlistV1.php" class="w3-bar-item w3-button w3-xxlarge material-icons">playlist_add</a>	
+					  <a href="contact.php" class="w3-bar-item w3-button w3-xxlarge material-icons" >contact_support</a>
+					  <a href="settings.php" class="w3-bar-item w3-button w3-xxlarge material-icons">settings</a>	
+    				</div>
 				</div>
 			</div>
-		</div>
-
-
+		
+		
 		<br>
 		<br>
 		<br>
 
 
-		<div class="w3-sidebar w3-bar-block w3-border-right" style="width:8%">
-			<a href="03_add_user(settings).php" class="w3-bar-item w3-button w3-xxlarge material-icons">person_add</a> 
-			<a href="02_show_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">visibility</a> 
-			<a href="04_update_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">manage_accounts</a>
-			<a href="05_delete_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">person_remove</a>
-		</div>
-
+			<!-- side navbar for admin + php codes -->
+			<div class="w3-sidebar w3-bar-block w3-border-right" style="width:8%">
+				<a href="03_add_user(settings).php" class="w3-bar-item w3-button w3-xxlarge material-icons">person_add</a> 
+				<a href="02_show_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">visibility</a> 
+			 	<a href="04_update_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">manage_accounts</a>
+			  	<a href="05_delete_user.php" class="w3-bar-item w3-button w3-xxlarge material-icons">person_remove</a>
 
-		<div style="margin-left:8%">
-
-			<div class="w3-container w3-dark-grey">
-				<h1>Show User</h1>
 			</div>
+		
 
-			<div class="w3-container">
-				<h5><user1>USERS</user1>
-					<password1>PASSWORDS</password1></h5>
-				<h6><section1>
-					<?php
-					//connect.php (tells where to connect servername, username, password, dbaseName)
-					require "communityproject_mysqli.php";
-
-					//create a variable to store sql code for the 'display all users' query
-					$query = ("SELECT * FROM users");
 
-					//run the query
-					$result = mysqli_query($conn,$query);
-
-					while ($output = mysqli_fetch_array($result))
-					{
-						echo "<user2>". $output['User_ID']."</user2><br>";
-						//closes the while loop
-					}
-					?>
-					</section1></h6>
-				<h6><section2>
-					<?php
-					//run the query
-					$result = mysqli_query($conn,$query);
+			<!-- Title for query: show users -->
+			<div style="margin-left:8%">
+				<div class="w3-container w3-dark-grey">
+					<h2>Show User</h2>
+				</div>
 
-					while ($output = mysqli_fetch_array($result))
-					{
-						echo "<user2>". $output['Password']."</user2><br>";
-						//closes the while loop
-					}
-					?>
-					</section2></h6>
+				
+				<!-- Users php -->
+				<div class="w3-row w3-panel w3-border">
+					<div class="w3-third w3-center w3-panel">
```

Comment:
added a border so the users and passwords aren't under each other and are next to one another

Reply:
this also makes it easier for the user - like Graeme to identify each user and their passwords